app/assets/stylesheets/config/_colors.scss

```diff
@@ -40,7 +40,7 @@
   --base-inverted: #fff;
 
   // Accent colors
-  --accent-brand: #3b49df;
+  --accent-brand: #3b49dd;
```

Comment:
This is almost exactly the same color as the original one. Does it actually fix any accessibility issues?

Reply:
I would guess it does. Sometimes little tweaks like this are all it takes for contrast issues. See https://webaim.org/resources/contrastchecker.

Reply:
Yea, I'm pretty sure I checked the contrast ratio for our main colors against some other main colors :D That's why I'm surprised and I'm wondering in which particular case this doesn't meet the standards :)

Reply:
It may pass over some backgrounds, but it doesn't in the footer. We could also introduce a different color variable for the footer links maybe? I've used Sass to lighten/darken colors programmatically before, could be an option.

Reply:
Oh, we don't use that color in the footer (i guess the right word to add here would be "anymore" :D) - links in the footer are simply black(-ish) now and on hover they turn into #323ebe which seems to pass contrast ratio test.